smartcontracts/contracts/test/MockOracle.sol

```diff
@@ -1,10 +1,10 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.4;
 
-import "@chainlink/contracts/src/v0.8/interfaces/ChainlinkRequestInterface.sol";
-import "@chainlink/contracts/src/v0.8/interfaces/LinkTokenInterface.sol";
-import "./LinkTokenReceiver.sol";
-import "./SafeMathChainlink.sol";
+import "@chainlink/contracts/src/v0.6/LinkTokenReceiver.sol";
+import "@chainlink/contracts/src/v0.6/interfaces/ChainlinkRequestInterface.sol";
+import "@chainlink/contracts/src/v0.6/interfaces/LinkTokenInterface.sol";
+import "@chainlink/contracts/src/v0.6/vendor/SafeMathChainlink.sol";
```

Comment:
@luislucena16 we can't use v0.6 version here because we're using Solidity 0.8 in the other contracts 🤔